engine/tests/classes/Elgg/Mocks/Database/Plugins.php

```diff
@@ -73,14 +73,20 @@ public function find($status = 'active') {
 		return $this->_plugins;
 	}
 
-	public function addTestingPlugin(ElggPlugin $plugin) {
+    public function generateEntities()
+    {
+        $this->addTestingPlugin(ElggPlugin::fromId('test_plugin', ''));
+        return true;
+    }
+
+    public function addTestingPlugin(ElggPlugin $plugin) {
```

Comment:
Wrong coding style. `{` needs to be on the same line as the function declaration.
Also are tabs used for spacing? On github it looks like the indentation is wrong.

Since tests are excluded from the code check this didn't come up.

Reply:
I had a lot of problems with the tabs-thing because my IDE was setup otherwise. But tabs should be set, yes. But I can see the wrong identation. Will look into it.